product_docs/docs/pgd/5/consistency/crdt/02_state-op-crdts.mdx

```diff
@@ -20,22 +20,26 @@ The main disadvantage is that you can't reset this value reliably in an asynchro
 
 !!! Note
     Implementing more complicated operation-based types by creating custom data types is possible, storing the state and the last operation. (Every change is decoded and transferred, so multiple
-    operations aren't needed). But at that point, the main benefits (simplicity, reuse of existing data types) are lost without gaining any advantage compared to state-based types (for example, still no capability to reset) except for the space requirements. (A per-node state isn't needed.)
+    operations aren't needed.) But at that point, the main benefits (simplicity, reuse of existing data types) are lost and no advantage gained compared to state-based types except for the space requirements. (A per-node state isn't needed.) For example, there's still no ability to reset. 
```

Comment:
Please check this edit.I was trying to break up a long sentence and too many parentheticals, and I wasn't 100% sure I got this right.

Reply:
Are you sure? If you take out the phrase "the main benefits...are lost", the sentence becomes "But at that point no advantage gained compared to state-based types except for the space requirements..." I believe the way "and" is functioning there it requires the "is" to make the sentence valid. ChatGPT agrees and says it is the only way to ensure subject/verb agreement.